### ISSUE TITLE
Distress / Emergency Wipe ?

### ISSUE THREAD

**Issue by macruspareto** [veracrypt/VeraCrypt#1016]
I use VC in full-OS encryption mode with a custom prompt ("Missing operating system") as a red herring.

Is there a way to wipe the bootsector and all masterkey data after 1 (deliberately wrong) password input if you're forced to turn on and boot your laptop? It would render it bricked but you still have the rescue ISO somewhere (maybe thousands of miles away) if you decide to restore it later.

**Comment by wendig0x** [comment]
I think that we can make a special password, which, when entered, destroys the header. This is good for an emergency. But this will not protect if the attacker gets hold of the HDD first, because he can replace the decryptor.

**Comment by macruspareto** [comment]
>> I think that we can make a special password, which, when entered, destroys the header. 

Good idea! I think of it as an anti-thug / anti-extortion layer of protection when travelling to "failed state" countries. 

It would require a commit to the bootloader code, wouldn't it? Too bad I'm not that fluent in x86 assembly :[
